velox/dwio/dwrf/test/DecoderUtilTest.cpp

```diff
@@ -14,15 +14,15 @@
  * limitations under the License.
  */
 
-#include "velox/dwio/dwrf/common/DecoderUtil.h"
+#include "velox/dwio/common/DecoderUtil.h"
```

Comment:
please also move this test to dwio/common

Reply:
moved